pkg/storaged/storage.scss

```
@@ -373,3 +373,13 @@ a.disabled {
     // 1em: margin between text and progress bar
     padding-inline-end: calc(var(--pf-v5-global--spacer--sm) + 3.5em + 1em) !important;
 }
+
+// FIXME: this makes the first card in anaconda mode inconsistent with other cards which have shadow
+// For fixing this propertly we would need to refactor the whole page rendering for anaconda mode
+.anaconda-mode {
+    .pf-v5-c-page__main-breadcrumb + .pf-v5-c-page__main-section {
+        .pf-v5-c-card {
+            box-shadow: none;
+        }
+    }
```

Would something like this work better? It visually merges the card using the PF margins and should handle mobile sizes too. Although we probably want to find out where that 576px magic number is coming from and use the SASS variable instead. (Sadly, CSS variables are not supported in media breakpoints yet.)

```suggestion
    .pf-v5-c-page__main-breadcrumb {
        margin: var(--pf-v5-c-page__main-section--PaddingTop) var(--pf-v5-c-page__main-section--PaddingRight) calc(-2 * var(--pf-v5-c-page__main-section--PaddingTop)) var(--pf-v5-c-page__main-section--PaddingLeft);
        // Remove the bottom of PF's box-shadow, but leave the top and sides intact.
        clip-path: rect(-10px -10px 100% calc(100% + 10px));
  
        @media screen and (max-width: 576px) {
              margin-inline: 0;
        }
    }
```